Pipeline/RunIDZCodeReview/README.md

```diff
@@ -14,6 +14,12 @@ This sample groovy `RunCodeReview.groogy` script
 - Generates an JCLExec for invoking the IDZ Code Review application
 - Stores the IDZ Code review reports ```CodeReviewCSV.csv```, ```CodeReviewJUNIT.xml```  as well as the JCL spool in the workdir.
 
+## Return codes
+The script will exists with the following return codes:
```

Comment:
I assume a little typo `exists` should be `exit`.

Reply:
It is now fixed. Thanks!